### ISSUE TITLE
Issue with markov order

### ISSUE THREAD

**Issue by martinjvickers** [martinjvickers/KAST#17]
With m=1, d2s, d2star (and d2) all give the exact same as Martins python code.
 
With m=2, the numbers diverge. D2s is a smaller divergence, but with d2star we get a negative number which should never happen.

**Comment by martinjvickers** [comment]
So, as an example;

```
$ cat seq1.fa seq2.fa 
>seq1
AGGCAGCGTACGAACCTACTGGAGTTGCGGTATGGGACCAGGCGACCTCTGATGCAGAGATACAGGAGCGCCGCGCCGGGTCTTCCTTGTAGAAGTCCTG
>seq2
CGGAGACCTCCGTGGACGGGGAAGTCCTGCGCGGGTCAGACGTACGCCCCGATTAGTTGCCCGGACGCCCGGTTGGCAGAAGTGACGGCGACTGCCCTCA
```

With KAST I get;

```
$ ./kast -q example_data/seq1.fa -r example_data/seq2.fa -t d2s -m 1 -k 3
Attempting to read the records 
############################ seq1
seq2 0.432464
```

```
$ /usr/bin/python ALFSC-contam-v002.py -q seq1.fa -r seq2.fa -t d2s -k 3 -m 1
...
Sorting scores for all references, no. of scores: 1
Number of scores 1
[[0.43246389442301436, 'seq2']]
```

But when I add `-m 2` to KAST I get;

```
$ ./kast -q example_data/seq1.fa -r example_data/seq2.fa -t d2s -m 2 -k 3
Attempting to read the records 
############################ seq1
seq2 0.124043

```
```
Number of scores 1
[[0.17143410528723957, 'seq2']]
```

**Comment by martinjvickers** [comment]
fixed with release 0.0.15
